.github/workflows/clean_packages.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
Suggest to rename file to "clean_up_packages_in_registry.yml"

Reply:
done